genienlp/arguments.py

```diff
@@ -652,6 +661,12 @@ def post_parse_train_specific(args):
             ' Otherwise the program (answer) will not be tokenized properly'
         )
 
+    if args.override_valid_metrics:
+        assert len(args.override_valid_metrics) == len(args.train_tasks) == len(args.val_tasks)
+        for train_task, val_task, metrics in zip(args.train_tasks, args.val_tasks, args.override_valid_metrics):
+            train_task.metrics = metrics
+            val_task.metrics = metrics
```

Comment:
Does this code work at all?
`override_valid_metrics` is a list of strings, so within the loop `metrics` is a single string, but `task.metrics` should be a list of strings.

Reply:
So `override_valid_metrics` is list of list of strings (note nargs='+' and action='append'). If you have multiple tasks, you would call this flag as many times.